gtsfm/configs/deep_front_end_astronet.yaml

```diff
@@ -0,0 +1,63 @@
+SceneOptimizer:
```

Comment:
maybe we could add a comment to the top of this yaml file, explaining that the only change w.r.t. `deep_front_end.yaml` is the RANSAC triangulation reproj threshold (10x looser)

Reply:
added